backend/utils.py

```diff
@@ -303,12 +305,12 @@ def offset_pagination(page, number_fetchs, query):
     try:
         number_fetchs = int(number_fetchs)
     except Exception:
-        number_fetchs = 5
+        number_fetchs = Utils.DEFAULT_PAGINATION_LIMIT
```

Comment:
esses 2 try/catch que tem aqui podem ser removidos se vc garantir uma conversão para inteiros logo na entrada da api, quando vc dá o get da query param. Você pode fazer uma função to_int ou algo do tipo, que converte para int e aí vc segue o fluxo do código sem ter que fazer try/catch, além de que except Exception é forçar a barra, pq pode ter outro problema e ser ignorado. Nunca é bom usar exception genérica.